src/ai/backend/manager/api/vfolder.py

```diff
@@ -450,7 +454,7 @@ async def create(request: web.Request, params: Any) -> web.Response:
         if group_uuid is not None:
             ownership_type = "group"
             quota_scope_id = QuotaScopeID(QuotaScopeType.PROJECT, group_uuid)
-            if not request["is_admin"]:
+            if not request["is_admin"] and group_type != ProjectType.MODEL_STORE:
```

Comment:
Is this about forbidding model vfolder creation in the model store by a non-admin user?
Previously, project vfolder creation was already forbidden for non-admin users.
Is this additional condition strictly necessary? I think it may be better to just add comments.